docs/accessibility-docs.md

```diff
@@ -0,0 +1,42 @@
+# Accessibility Docs
+
+If you are looking for accessibility docs[^1] for
```

Comment:
Well this is annoying. When I build the docs locally, the footnote works, but when I look at this page on the ReadTheDocs preview build, it doesn't format the footnote: https://jupyter-accessibility--123.org.readthedocs.build/en/123/accessibility-docs.html